<launch>
  <arg name="robot_namespace" default="robot"/>

  <group ns="$(arg robot_namespace)">
    <node name="backup_depth_publisher" pkg="artifact_localization" type="topic_repeater.py" output="screen">
    </node>
  </group>

</launch>
